src/main/java/org/deepsymmetry/beatlink/CdjStatus.java

```diff
@@ -642,6 +643,8 @@ public CdjStatus(DatagramPacket packet) {
         if (Util.isOpusQuad(deviceName)) {
             trackSourcePlayer = translateOpusPlayerNumbers(packetBytes[40]);
             trackSourceSlot = findOpusTrackSourceSlot();
+            // isLocalSdLoaded() == true
+            packetBytes[115] = 0;
```

Comment:
We’ll want this to seem like `isLocalUsbLoaded()` is `true` also, and I am not sure what we are going to do about representing data references to tracks in USB 3 yet, we’ll have to think about that a bit. And I think this code should not always assume it is loaded, it should set it to true or false depending on whether the `OpusProvider` has a metadata archive attached to that slot, right?

Reply:
If we set isLocalUsbLoaded() to true as well that will cause MetadataFinder.handleUpdate to recordMount twice, once for each type of slot. If thats ok then I would be happy to set isLocalUsbLoaded() as well.

Reply:
Right, I think what will happen is it becomes true and false for each slot as status packets come in and we have attached or detached metadata archives for those slots, so the mounts and unmounts will be reported in response to those attach/detach actions on the part of the user. Does that make sense?

Reply:
Makes sense. Setting it up!